invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/DocumentsDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -13,17 +13,32 @@ import {
 import PropTypes from 'prop-types';
 import DocumentTab from '../DocumentTab';
 import { RequestNewLoanForm } from './components/RequestNewLoanForm';
+import isEmpty from 'lodash/isEmpty';
 import '../../DocumentsDetails.scss';
+import { goTo } from '../../../../../history';
+import { FrontSiteRoutes } from '../../../../../routes/urls';
+import { document as documentApi } from '../../../../../common/api';
 
 export default class DocumentMetadata extends Component {
   constructor(props) {
     super(props);
     this.document = this.props.documentsDetails;
+    this.viewSeriesUrl = FrontSiteRoutes.documentsListWithQuery;
   }
   state = { visible: false };
 
   toggleVisibility = () => this.setState({ visible: !this.state.visible });
 
+  goToSeriesList = seriesPid =>
+    goTo(
+      this.viewSeriesUrl(
```

Comment:
Referring to my comment https://github.com/inveniosoftware/invenio-app-ils/pull/371/files#r292394280, use `FrontSiteRoutes.documentsListWithQuery`